api/v1beta1/event.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "api/v1beta1/event_data.proto";
+
+message Event {
+    google.protobuf.Timestamp timestamp = 1;
+    uint32 id = 2;
+    string name = 3;
+    Policies policies = 4;
+    Context context = 5;
+    map<string, EventValue> event_data = 6;
+    optional Threat threat = 7;
+}
+
+message Policies {
+    repeated string matched = 1;
+}
+
+message Context {
+    optional Process process = 1;
+    optional Container container = 2;
+    optional K8s k8s = 3;
+}
+
+message Process  {
+    Executable executable = 1;
+    google.protobuf.UInt32Value entity_id = 2;
+    google.protobuf.UInt32Value pid = 3;
+    google.protobuf.UInt32Value namespaced_pid = 4;
+    User real_user = 5;
+    Thread thread = 6;
```

Comment:
Since Process is now also used for Parent, we should add optional for:
Executable
User
Thread